frontend/src/wizards/Argo/ArgoWizard.tsx

```diff
@@ -651,28 +671,37 @@ export function DeploymentWindow(props: { timeZone: string[] }) {
 }
 
 export function TimeWindow(props: { timeZone: string[] }) {
+    const { t } = useTranslation()
     return (
         <Stack hasGutter style={{ paddingBottom: 16 }}>
             <WizMultiSelect
-                label="Time window configuration"
-                placeholder="Select at least one day to create a time window."
+                label={t('Time window configuration')}
+                placeholder={t('Select at least one day to create a time window.')}
                 path="timewindow.daysofweek"
                 required
-                options={['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday']}
+                options={[
+                    t('Monday'),
+                    t('Tuesday'),
+                    t('Wednesday'),
+                    t('Thursday'),
+                    t('Friday'),
+                    t('Saturday'),
+                    t('Sunday'),
+                ]}
```

Comment:
Careful here as well to distinguish values vs. displayed values

Reply:
will revert to untranslated strings for now

Reply:
reverting

Reply:
If you reverted, the change has not been pushed. Marking as unresolved.

Reply:
up now